frameworks/hdfs/tests/config.py

```diff
@@ -56,15 +56,14 @@ def get_active_name_node(service_name):
     raise Exception("Failed to determine active name node")
 
 
+@retrying.retry(
+    wait_fixed=10000,
+    stop_max_delay=DEFAULT_HDFS_TIMEOUT*1000)
 def get_name_node_status(service_name, name_node):
-    def get_status():
-        rc, output = run_hdfs_command(service_name, "./bin/hdfs haadmin -getServiceState {}".format(name_node))
-        if not rc:
-            return rc
-
-        return output.strip()
-
-    return shakedown.wait_for(lambda: get_status(), timeout_seconds=DEFAULT_HDFS_TIMEOUT)
+    rc, output = run_hdfs_command(service_name, "./bin/hdfs haadmin -getServiceState {}".format(name_node))
+    if not rc:
+        return rc
+    return output.strip()
```

Comment:
Is a blank string successful output? Would retry not treat this as failure?

Reply:
Good catch! this is an inconsistency in the refactor. See above for fix